ComponentKit/Utilities/CKComponentAction.mm

```diff
@@ -296,6 +296,59 @@ void _CKTypedComponentDebugCheckTargetSelector(id target, SEL selector, const st
 
 @end
 
+#pragma mark - Accessibility Actions
+
+@interface WFComponentAccessibilityCustomAction : UIAccessibilityCustomAction
+- (instancetype)initWithName:(NSString *)name action:(CKComponentAction)action view:(UIView *)view;
+@end
+
+@implementation WFComponentAccessibilityCustomAction
+{
+  UIView *_ck_view;
+  CKComponentAction _ck_action;
+}
+
+- (instancetype)initWithName:(NSString *)name action:(CKComponentAction)action view:(UIView *)view
+{
+  if (self = [super initWithName:name target:self selector:@selector(ck_send)]) {
+    _ck_view = view;
+    _ck_action = action;
+  }
+  return self;
+}
+
+- (BOOL)ck_send
+{
+  _ck_action.send(_ck_view.ck_component, CKComponentActionSendBehaviorStartAtSender);
+  return YES;
+}
+
+@end
+
+CKComponentViewAttributeValue CKComponentAccessibilityCustomActionsAttribute(const std::vector<std::pair<NSString *, CKComponentAction>> &passedActions)
+{
+  auto const actions = std::move(passedActions);
```

Comment:
Just trying to copy `actions` so that the block properly captures it (as opposed to an invalid reference). I changed this to just assign instead!